fun_with_flags/translations/tr/LC_MESSAGES/messages.po

```diff
@@ -397,11 +418,15 @@ msgid ""
 "          You're in the\n"
 "          unicorn club. <br><i>Received"
 msgstr ""
+"\n"
+"        <b>Kulüp:</b>\n"
+"          Sen içindesin\n"
+"          unicorn kulübü. <br><i>Alındı</i>"
```

Comment:
you shouldn't close the i tag here (</i>) it's closed later in the template. this is true for all similar for all three badges.

Reply:
fixed

Reply:
cool, but there's the same issue sstill on lines 441 & 457. please also fix there. cheers!

Reply:
fixed again.